### PR TITLE
{chem}[foss/2020a,intel/2020a] PLUMED v2.6.0 w/ Python 3.8.2

### PR DESCRIPTION
**Pull request by ocaisa**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by ocaisa** [comment]
Test report by @ocaisa
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/551fe2b3fceb1719bc2ba5061e7ed149 for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 6 out of 6 (2 easyconfigs in this PR)
login2.cerberus.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6126 CPU @ 2.60GHz, Python 2.7.5
See https://gist.github.com/b4d688a426d690b90d53bb1c03459110 for a full test report.

File: easybuild/easyconfigs/p/PLUMED/PLUMED-2.6.0-foss-2020a-Python-3.8.2.eb [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+]
+
+preconfigopts = 'env FC=$MPIF90 LIBS="$LIBLAPACK $LIBS" '
+configopts = ' --exec-prefix=%(installdir)s --enable-gsl --enable-modules=all --enable-python'
+configopts += ' --enable-boost_graph --enable-boost_serialization '
+configopts += ' --enable-asmjit '
+prebuildopts = 'source sourceme.sh && '
+
+# make sure that ld.gold linker is used
+# requires to work around problems like "ld: BFD (GNU Binutils) 2.30 assertion fail elf.c:3564"
```
Comment (akesandgren) [inline easybuild/easyconfigs/p/PLUMED/PLUMED-2.6.0-foss-2020a-Python-3.8.2.eb]: And "required", should probably also state that this appliles to the intel build, but for simplicity we keep them identical

**Comment by ocaisa** [comment]
Test report by @ocaisa
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/51a2dd342e1c687f006a61615a2a0bd3 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/5ae57793c20ba45134159861b56e1cad for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 11 out of 11 (2 easyconfigs in this PR)
node3116.skitty.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/e50ca567478488dcaf2477f8e5739396 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 10 out of 10 (2 easyconfigs in this PR)
node2414.golett.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/0a5b9f83c45b62de5c5031219f3426fe for a full test report.

**Comment by boegel** [comment]
Going in, thanks @ocaisa!

**Pull request merged**